codegen/templates/tchannel_endpoint.tmpl

```diff
@@ -100,6 +116,7 @@ func (h *{{$handlerName}}) Handle(
 
 	{{if eq (len .Exceptions) 0 -}}
 		if err != nil {
+			h.Logger.Error("Handler returned error", zap.Error(err))
```

Comment:
not an `Error()` ; do we log about this anywhere else ?

Reply:
Changed to warning.  This is the only place where we log errors returned by `workflow.Handle()`.  The `workflow.Handle()` function itself is provided by the developer, so may or may not log the error it returns.